### PR TITLE
Use jackson to replace the related code of fastjson, and improve the related unit test of jackson

### PR DESCRIPTION
**Pull request by 329724814**
## Why are the changes needed?

For project security, fastjson needs to be replaced with jackson. First, a jackson tool class is needed to encapsulate some functions of fastjson

*(For example: The Arctic Flink dataStream API has already supported this configuration: 'scan.startup.mode', but it is not available with Flink SQL. We should expose this configuration to users when they are using Flink SQL.)*

## Brief change log

*(For example:)*
  - *Add the 'scan.startup.mode' configuration to the ArcticScanContext in the flink modules*
  - *The flink 1.12 and 1.14 versions are affected.*
  - *Add the documentation in the flink chapter*

## How was this patch tested?
- [ x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x ] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/JacksonUtils.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        JavaType javaType = OBJECT_MAPPER.getTypeFactory().constructParametricType(ArrayList.class, String.class);
+        try {
+            return OBJECT_MAPPER.readValue(jsonString, javaType);
+        } catch (JsonProcessingException e) {
+            throw new IllegalStateException(DESERIALIZE_ERROR, e);
+        }
+    }
+
+    public static String getString(JsonNode jsonNode, String fieldName){
+        if(jsonNode == null || jsonNode.get(fieldName) == null){
```
Comment (shidayang) [inline ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/JacksonUtils.java:89]: jsonNode is null should through exception or make a args to control

File: ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/TypeUtils.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @Author alex.wang
+ * @Date 2023/2/28 17:19
+ * @PackageName: com.netease.arctic.ams.server.utils
+ * @Version 1.0
+ */
```
Comment (shidayang) [inline ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/TypeUtils.java:16]: Please remove the head

File: ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/TypeUtils.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+public class TypeUtils {
```
Comment (shidayang) [inline ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/TypeUtils.java:17]: TypeCastUtil would be better?

File: ams/ams-server/src/test/java/com/netease/arctic/ams/server/utils/JacksonUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        }
+    }
+
+    @Test
+    public void testJacksonUtils() throws Exception{
+        Student wangxiao = new Student();
+        wangxiao.setAge(20);
+        wangxiao.setName("WangXiao");
+        wangxiao.setChineseStudent(true);
+        wangxiao.setTimeStamp(System.currentTimeMillis());
```
Comment (shidayang) [inline ams/ams-server/src/test/java/com/netease/arctic/ams/server/utils/JacksonUtilsTest.java:67]: wangxiao?

File: ams/ams-server/src/test/java/com/netease/arctic/ams/server/JacksonTest.java [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+ * @PackageName: com.netease.arctic.ams.server
+ * @Version 1.0
+ */
+public class JacksonTest {
+
+
+
+    @Test
+    public void jacksonTest(){
+        String configPath = System.getProperty("user.dir") + "/src/test/resources/test.yaml";
```
Comment (shidayang) [inline ams/ams-server/src/test/java/com/netease/arctic/ams/server/JacksonTest.java:30]: Maybe using string be better than file

**Review by shidayang — CHANGES_REQUESTED** [review]

**Comment by shidayang** [comment]
Thanks for your contributions. All class must have license

**Comment by shidayang** [comment]
This pr relate a issue, please label "fix #xxx" in comment

**Comment by shidayang** [comment]
The title format does not conform to the specification
